test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/Extensions/Binders/ApplyBinderTests.cs

```diff
@@ -59,7 +59,7 @@ public void BindApplyWithAggregateShouldReturnApplyClause()
             aggregate.Expressions.Should().HaveCount(1);
 
             var statements = aggregate.Expressions.ToList();
-            var statement = statements[0];
+            var statement = statements[0] as PropertyAggregateExpression;

```

Comment:
Assert statement is not null after this fail test as opposed to crash with null ref exception.